zebra/zebra_nb.c

```diff
@@ -0,0 +1,2498 @@
+ */
+struct yang_data *
+lib_vrf_ribs_rib_route_nexthop_group_entry_is_recursive_get_elem(
+	const char *xpath, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/is-onlink
+ */
+struct yang_data *lib_vrf_ribs_rib_route_nexthop_group_entry_is_onlink_get_elem(
+	const char *xpath, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/is-active
+ */
+struct yang_data *lib_vrf_ribs_rib_route_nexthop_group_entry_is_active_get_elem(
+	const char *xpath, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/mpls-label-stack/entry
+ */
+const void *
+lib_vrf_ribs_rib_route_nexthop_group_entry_mpls_label_stack_entry_get_next(
+	const void *parent_list_entry, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+int lib_vrf_ribs_rib_route_nexthop_group_entry_mpls_label_stack_entry_get_keys(
+	const void *list_entry, struct yang_list_keys *keys)
+{
+	/* TODO: implement me. */
+	return NB_OK;
+}
+
+const void *
+lib_vrf_ribs_rib_route_nexthop_group_entry_mpls_label_stack_entry_lookup_entry(
+	const void *parent_list_entry, const struct yang_list_keys *keys)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/mpls-label-stack/entry/id
+ */
+struct yang_data *
+lib_vrf_ribs_rib_route_nexthop_group_entry_mpls_label_stack_entry_id_get_elem(
+	const char *xpath, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/mpls-label-stack/entry/label
+ */
+struct yang_data *
+lib_vrf_ribs_rib_route_nexthop_group_entry_mpls_label_stack_entry_label_get_elem(
+	const char *xpath, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/mpls-label-stack/entry/ttl
+ */
+struct yang_data *
+lib_vrf_ribs_rib_route_nexthop_group_entry_mpls_label_stack_entry_ttl_get_elem(
+	const char *xpath, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath:
+ * /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/mpls-label-stack/entry/traffic-class
+ */
+struct yang_data *
+lib_vrf_ribs_rib_route_nexthop_group_entry_mpls_label_stack_entry_traffic_class_get_elem(
+	const char *xpath, const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
+
+/*
+ * XPath: /frr-vrf:lib/vrf/frr-zebra:ribs/rib/route/nexthop-group/entry/mtu
+ */
+struct yang_data *
+lib_vrf_ribs_rib_route_nexthop_group_entry_mtu_get_elem(const char *xpath,
+							const void *list_entry)
+{
+	/* TODO: implement me. */
+	return NULL;
+}
```

Comment:
To improve organization, I think it would be better to move all callbacks to files like _zebra_nb_config.c_, _zebra_nb_state.c_, etc. and keep only the _frr_zebra_info_ structure in this file (just like was done in the other daemons recently).